react/features/mobile/audio-mode/middleware.js

```diff
@@ -140,7 +140,7 @@ function _updateAudioMode({ getState }, next, action) {
     const { enabled: audioOnly } = state['features/base/audio-only'];
     let mode;
 
-    if (Platform.OS === 'android' && getFeatureFlag(state, ANDROID_AUDIO_FOCUS_DISABLED, false)) {
+    if (getFeatureFlag(state, AUDIO_FOCUS_DISABLED, false)) {
```

Comment:
instead of doing this, just check if the flag is enabled an return early, so we never set the mode. Setting the mode changes the category on iOS.